typespec-extension/src/code-model-builder.ts

```diff
@@ -673,17 +715,17 @@ export class CodeModelBuilder {
    * @param versions api-versions to filter
    * @returns filtered api-versions
    */
-  private getFilteredApiVersions(
-    pinnedApiVersion: Version | undefined,
-    versions: Version[],
+  private getFilteredApiVersionsFromString(
```

Comment:
Similar, if you do not have another function of `getFilteredApiVersionsFrom<Type>`, just use simpler name.

The type is on the parameter.

Reply:
See the reply here: https://github.com/Azure/autorest.java/pull/2861#discussion_r1713462201